### PR TITLE
Move model resources to recommended patern.

### LINKED ISSUE #176: Mvc.LocalizationSample.Web Model resources don't follow patern
The resources for Mvc.LocalizationSample.Web models don't follow our recomended model of the resource key being the translation of the default language, change them so they match this model like the Controller and View resources do.

### PR DESCRIPTION
**Pull request by ryanbrandenburg**
Fixes #176 by moving the model resources to our recommended pattern (key as default language).

### CODE REVIEW

File: samples/Mvc.LocalizationSample.Web/Models/Product.cs [hunk 1]
```diff
@@ -7,10 +7,10 @@ namespace Mvc.LocalizationSample.Web.Models
 {
     public class Product
     {
-        [Required(ErrorMessage = "ProductName")]
+        [Required(ErrorMessage = "Product Name is invalid")]
```
Comment (dougbu) [inline samples/Mvc.LocalizationSample.Web/Models/Product.cs]: `"{0} is invalid."`

Use `[Display(Name = "Product Name")]` to change the name shown. Add resources for `"Product Name"` if that should also be localized. Similar comments in other places where you're not displaying the property name.

File: samples/Mvc.LocalizationSample.Web/Models/Product.cs [hunk 2]
```diff
@@ -7,10 +7,10 @@ namespace Mvc.LocalizationSample.Web.Models
...
         public string ProductName { get; set; }
 
-        [MinLengthSix(ErrorMessage = "MinLengthSix")]
+        [MinLengthSix(ErrorMessage = "description must be at least 6 characters")]
```
Comment (dougbu) [inline samples/Mvc.LocalizationSample.Web/Models/Product.cs]: `"{0} must be at least 6 characters."`

File: samples/Mvc.LocalizationSample.Web/Models/User.cs [hunk 3]
```diff
@@ -7,7 +7,7 @@ namespace Mvc.LocalizationSample.Web.Models
 {
     public class User
     {
-        [MinLength(6, ErrorMessage = "NameError")]
+        [MinLength(6, ErrorMessage = "Invalid name. name minimum length is {1}")]
```
Comment (dougbu) [inline samples/Mvc.LocalizationSample.Web/Models/User.cs]: `"Invalid {0}. {0} minimum length is {1}."`

**Comment by ryanbrandenburg** [comment]
:up::date:

File: samples/Mvc.LocalizationSample.Web/Models/Product.cs [hunk 4]
```diff
@@ -7,12 +7,15 @@ namespace Mvc.LocalizationSample.Web.Models
 {
     public class Product
     {
-        [Required(ErrorMessage = "ProductName")]
+        [Display(Name = "Name")]
```
Comment (dougbu) [inline samples/Mvc.LocalizationSample.Web/Models/Product.cs]: 😢 I liked "Product Name". More importantly this and a `User`'s name now have the same label in the generated HTML. Suggest going back to "Product Name" here to remove that ambiguity.

**Pull request merged**